### ISSUE TITLE
Must use df.geometry not df['geometry'] when specifying column geometry

### ISSUE THREAD

**Issue by nejohnson2** [geopandas/geopandas#182]
After performing a merge, the GeoDataFrame did not recognize the geometry column as a GeoSeries column.  (related to issue 181).  In order to reset the geometry one must use df.geometry and not df['geometry'], though I believed these two methods to be the same.  Any reason df['geometry'] doesnt work?

**Comment by jwass** [comment]
@nejohnson2 Yes, this looks like the same issue as #181 and #127. But only if you're using `pd.merge(...)` instead of the `merge` method on the GeoDataFrame directly.
